src/parsers/babel_parser.js

```diff
@@ -171,3 +170,11 @@ export const parse = (file: string, data: ?string): ParseResult => {
 
   return parseResult;
 };
+
+export const plugins = ['jsx', 'classProperties'];
+
+// Its not possible to use the parser with flow and typescript active at the same time
+export const parseJs = (file: string, data: ?string, additionalPlugins: string[] = []): ParseResult =>
+  parse(file, data, {plugins: [...plugins, ...additionalPlugins, 'flow']});
+export const parseTs = (file: string, data: ?string, additionalPlugins: string[] = []): ParseResult =>
+  parse(file, data, {plugins: [...plugins, ...additionalPlugins, 'typescript']});
```

Comment:
i provided a more solid list of plugins. (for feature i see used more regularly).
With spreading those plugins together the consumer is able to add addtional plugins.
i don't see why anyone would want to remove plugins, this would require a different implementation. maybe like: use the alternativePlugins instead of plugins if provided. but that would probably be more prone to errors.